backends/events/sdl/sdl-events.cpp

```diff
@@ -85,6 +117,8 @@ SdlEventSource::SdlEventSource()
 		if (SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER) == -1) {
 			error("Could not initialize SDL: %s", SDL_GetError());
 		}
+		else
```

Comment:
That else statement is not necessary as `error` quits ScummVM if `SDL_InitSubSystem` fails.